tracing-api/src/main/java/com/palantir/tracing/api/Span.java

```diff
@@ -16,17 +16,13 @@
 
 package com.palantir.tracing.api;
 
-import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
-import com.fasterxml.jackson.databind.annotation.JsonSerialize;
 import java.util.Map;
 import java.util.Optional;
 import org.immutables.value.Value;
 
 /**
  * A value class representing a completed Span, see {@link OpenSpan} for a description of the fields.
  */
-@JsonDeserialize(as = ImmutableSpan.class)
-@JsonSerialize(as = ImmutableSpan.class)
 @Value.Immutable
 @Value.Style(visibility = Value.Style.ImplementationVisibility.PACKAGE)
 public abstract class Span {
```

Comment:
I don't know why this included jackson annotations to begin with other than copy/paste from another immutables type. @uschi2000, @markelliot, @ellisjoe are you aware of anything that intentionally deserializes tracing-api spans from json?

Reply:
Forgot the exact details, but I suspect that the initial tracing implementation may have serialized this class directly instead of going via `ZipkinCompatSpan` in `AsyncSlf4jSpanObserver`. I am not aware of anyone serializing or deserializing via this class, but I also don't have exhaustive knowledge of the world, obviously. It's *probably* fine to remove, but I can't make a certain determination.